.cloud-gov/manifest-dev.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
per this, were you hoping to have this push via CI/CD in a certain tagged branch or you're still planning on manually deploying to the dev instance?

Reply:
For now, I will just push manually to this dev server when we want to.

I'd like to get to the place where all PRs get their own deployment automatically. Greg said he's done that before with cloud.gov. The PR servers are great for sharing ideas and work in progress.